content/01.abstract.md

```diff
@@ -10,4 +10,4 @@ Manubot is described in the paper titled "Open collaborative writing with Manubo
 
 Test my understanding
 
-Just a paper [@url:https://arxiv.org/abs/1910.11619]
+Just a paper [@url:https://www.nature.com/articles/nature18003]
```

Comment:
I would recommend a DOI citation for this (or pmid, or other) but happy to merge as is :)
```suggestion
Just a paper [@doi:10.1038/nature18003]
```